src/main/java/com/salesforce/storm/spout/dynamic/VirtualSpout.java

```diff
@@ -256,6 +256,8 @@ public void close() {
      */
     @Override
     public Message nextTuple() {
+        getMetricsRecorder().assignValue(getClass(), getVirtualSpoutId() + ".filters", getFilterChain().getSteps().size());
```

Comment:
@Crim This seemed as good a place as any to put this, but I'm open to other suggestions.

Reply:
This feels like we're adding a metric to the critical code path that won't change often.

Reply:
Agreed, where’s a better place to stick it then?